app/views/students/student_report.pdf.erb

```diff
@@ -66,8 +66,8 @@
         <ul>
           <li>DOB: <%= format_date(@student.date_of_birth) %></li>
           <li><%= "Grade #{@student.grade}" %></li>
-          <li><%= @student.try(:school).try(:name) %></li>
-          <li><%= @student.try(:homeroom).try(:name) %></li>
+          <li><%= @student.school.name || "N/A" %></li>
```

Comment:
If the student isn't assigned to a school, this will blow up with:

```
NoMethodError: undefined method `name' for nil:NilClass
```

We still want to be able to generate the report if the student isn't assigned to a school, so I think some sort of `try` here is useful. Instead of "N/A" maybe we want "No School Assigned" since there's no other way for the reader to tell what attribute of the student is "N/A".

Reply:
That makes sense, should I make that change for homeroom as well?

Reply:
@gabivoicu Yep that sounds right to me!